test/cmd/newapp.sh

```diff
@@ -126,6 +158,10 @@ os::cmd::expect_success 'oc delete template ruby-helloworld-sample'
 os::cmd::expect_success_and_text 'oc new-app mysql --name=db' 'db'
 os::cmd::expect_success 'oc new-app https://github.com/openshift/ruby-hello-world -l app=ruby'
 os::cmd::expect_success 'oc delete all -l app=ruby'
+# check for error when template JSON file has errors
+jsonfile="${TMPDIR:-/tmp}/foo.json"
+echo '{"test"}' > "${jsonfile}"
```

Comment:
I don't expect it is - we simply have to have a file on disk somewhere.  I
may just move this to a fixture.

On Tue, Feb 2, 2016 at 3:41 AM, Rodolfo Carvalho <notifications@github.com>
wrote:

> In test/cmd/newapp.sh
> <https://github.com/openshift/origin/pull/6670#discussion_r51538538>:
>
> > @@ -126,6 +158,10 @@ os::cmd::expect_success 'oc delete template ruby-helloworld-sample'
> >  os::cmd::expect_success_and_text 'oc new-app mysql --name=db' 'db'
> >  os::cmd::expect_success 'oc new-app https://github.com/openshift/ruby-hello-world -l app=ruby'
> >  os::cmd::expect_success 'oc delete all -l app=ruby'
> > +# check for error when template JSON file has errors
> > +jsonfile="${TMPDIR:-/tmp}/foo.json"
> > +echo '{"test"}' > "${jsonfile}"
>
> I searched for TMPDIR in the project, but didn't find any place that
> would set it to anything other than /tmp.
> I'm assuming this file will not be deleted by the test framework after
> run, is it to be considered a problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6670/files#r51538538>.
>